sdk/core/azure-core-test/src/main/java/com/azure/core/test/annotation/DoNotRecord.java

```diff
@@ -27,6 +23,9 @@
      * Returns whether the test will be ignored during a {@link TestMode#PLAYBACK playback} test run.
      *
      * @return Flag indicating if the test will be ignored during a playback test run.
+     * @deprecated If a test should not run in playback, use {@link LiveOnly} instead. This will be removed in a future
+     * release.
      */
+    @Deprecated
```

Comment:
Is this still used anywhere? I guess this is technically public surface area but I would like to think our test library is a place where we could just remove this without a deprecation cycle.

Reply:
It is still being used and I'd want to see in another change how those places are being used as this has always been a bit of an odd configuration.